en/creating-an-artboard.md

```diff
@@ -18,45 +18,45 @@ At the end of this topic you'll have a Sketch Artboard for a registration screen
+    - **1-5 Pages**  
+      All the styling assets such as Material Icons, Colors, Typography, Elevation, and Illustrations.
 
-    - **Indigo-Components**  
+    - **6 Page**  
       A vast set of Components to layout your design ideas.  
       Through the overrides section you can choose from numerous presets compatible with Ignite UI for Angular and our code generation engine.
 
-    - **Indigo-Patterns**  
-      The library where components shape up meaningful patterns for productive application design.
+    - **7-26 Pages**  
+       The components shape up meaningful patterns for productive application design.
 
 2.  Open up the Sketch app on your Mac, click on the Sketch item from the Apple Menu, and select Preferences.
 
 3.  In the dialog, select the **Libraries** tab and click the **Add Library...** button.
 
-4.  Navigate to the folder where you have downloaded the Libraries.
+4.  Navigate to the folder where you have downloaded the library.
 
-5.  Select all three of them and click **Open**.
+5.  Select it and click **Open**.
 
-6.  Make sure they show up in the list in the **Preferences** dialog and that they all are checked.
+6.  Make sure it shows up in the list in the **Preferences** dialog and that it is checked.
 
   <div class="divider--half"></div>
-  <img class="responsive-img" src="images/creating_an_artboard_2.png" />
+  <img class="responsive-img" src="images/Creating_An_Artboard_2.png" srcset="images/Creating_An_Artboard_2@2x.png" />
   <div class="divider--half"></div>
   <div class="divider--half"></div>
   <div class="divider--half"></div>
   <div class="divider--half"></div>
   <div class="divider--half"></div>
 
-You may have noticed that in the image above the libraries also appear to be present in my Sketch cloud, and as an alternative, you can use this mechanism in Sketch if you are running version 57 or later. The set up is very easy, you only need to open Styling, Components, and Patterns one after another and save each file in your Sketch cloud. A detailed explanation of how to set Indigo.Design as cloud libraries can be found in [this Medium post](https://medium.com/ignite-ui/using-indigo-design-with-sketch-cloud-libraries-70533bca00f8). Once you added the Libraries successfully, they will show up in the **Symbols** section of the **Insert** menu in Sketch.
+You may have noticed that in the image above the library also appear to be present in my Sketch cloud, and as an alternative, you can use this mechanism in Sketch if you are running version 57 or later. The set up is very easy, you only need to open Indigo.Design and save it in your Sketch cloud. A detailed explanation of how to set Indigo.Design as cloud library can be found in [this Medium post](https://medium.com/ignite-ui/using-indigo-design-with-sketch-cloud-libraries-70533bca00f8). Once you added the library successfully, it will show up in the **Symbols** section of the **Insert** menu in Sketch.
```

Comment:
```suggestion
You may have noticed that in the image above the library also appears to be present in my Sketch cloud, and as an alternative, you can use this mechanism in Sketch if you are running version 57 or later. The set up is very easy, you only need to open the Indigo.Design Sketch file and save it in your Sketch cloud. A detailed explanation of how to set Indigo.Design as a cloud library can be found in [this Medium post](https://medium.com/ignite-ui/using-indigo-design-with-sketch-cloud-libraries-70533bca00f8). Once you added the library successfully, it will show up in the **Symbols** section of the **Insert** menu in Sketch.
```